mars/tensor/utils.py

```diff
@@ -554,9 +556,13 @@ def create_fetch_tensor(chunk_size, shape, dtype, tensor_key=None, tensor_id=Non
     for chunk_shape, chunk_idx, chunk_key in izip(itertools.product(*chunk_size),
                                                   itertools.product(*chunk_size_idxes),
                                                   chunk_keys):
+        if chunk_key is None:
+            chunk_key = tokenize(uuid.uuid4().hex)
```

Comment:
I think you can just add some extra property to `new_chunk`, e.g. `new_chunk(None, shape=xx, index=xx, hex=uuid.uuid4().hex)`, which would make chunk key different.

Reply:
Fixed.